spec/features/idv/doc_auth/how_to_verify_spec.rb

```diff
@@ -4,30 +4,31 @@
   include IdvHelper
   include DocAuthHelper
 
-  context 'opt-in ipp is turned on' do
-    let(:enabled) { true }
+  let(:enabled) { true }
 
-    before do
-      allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { enabled }
+  before do
+    allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { enabled }
 
-      sign_in_and_2fa_user
-      complete_doc_auth_steps_before_agreement_step
-      complete_agreement_step
-    end
+    sign_in_and_2fa_user
+    complete_doc_auth_steps_before_agreement_step
+    complete_agreement_step
+  end
 
-    context 'opt-in ipp is turned off' do
-      let(:enabled) { false }
+  context 'opt-in ipp is turned off' do
+    let(:enabled) { false }
 
-      it 'skips when disabled' do
-        expect(page).to have_current_path(idv_hybrid_handoff_url)
-      end
+    it 'skips when disabled' do
+      expect(page).to have_current_path(idv_hybrid_handoff_url)
     end
+  end
 
+  context 'opt-in ipp is turned on' do
```

Comment:
I have in the past done stuff like duplicate `let(:enabled) { true }` in a context like this to make it clear, even though it's already the default value. But not everyone was a fan. 😄 Just sharing as a completely optional idea.